code/XFileExporter.cpp

```diff
@@ -514,12 +514,16 @@ void XFileExporter::WriteMesh(aiMesh* mesh)
 
 std::string XFileExporter::toXFileString(aiString &name)
 {
-	std::string str = std::string(name.C_Str());
-	std::replace(str.begin(), str.end(), '<', '_');
-	std::replace(str.begin(), str.end(), '>', '_');
-	std::replace(str.begin(), str.end(), '{', '_');
-	std::replace(str.begin(), str.end(), '}', '_');
-	std::replace(str.begin(), str.end(), '$', '_');
+	std::string pref = "NN_"; // node name prefix to prevent unexpected start of string
```

Comment:
I don't like such a prefix. If we re-import it, the scene would be changed. Why can it not start with _ - does the X file spec forbit it and it would not parse again?